### PR TITLE
Run codespell

### PR DESCRIPTION
**Pull request by rhatdan**
[NO NEW TESTS NEEDED]

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by flouthoc** [comment]
LGTM but PR needs a rebase.

File: pkg/specgen/specgen.go [hunk 1]
```diff
@@ -423,7 +423,7 @@ type ContainerNetworkConfig struct {
 	// PublishExposedPorts is set.
 	// Optional.
 	Expose map[uint16]string `json:"expose,omitempty"`
-	// Map of networks names ot ids the container should join to.
+	// Map of networks names to ids the container should join to.
```
Comment (Luap99) [inline pkg/specgen/specgen.go:426]: This should be `or`

File: pkg/auth/auth_test.go [hunk 2]
```diff
@@ -31,7 +31,7 @@ var largeAuthFileValues = map[string]types.DockerAuthConfig{
 
 // systemContextForAuthFile returns a types.SystemContext with AuthFilePath pointing
 // to a temporary file with fileContents, or nil if fileContents is empty; and a cleanup
-// function the calle rmust arrange to call.
+// function the called rmust arrange to call.
```
Comment (mheon) [inline pkg/auth/auth_test.go:34]: Should be caller, not called. And there's an extra r on the must

Comment (rhatdan) [reply pkg/auth/auth_test.go]: Strange that codespell missed that.

**Comment by TomSweeneyRedHat** [comment]
LGTM
once test still not happy, I'll rekick it.

**Pull request merged**